docs/modules/ROOT/pages/features.adoc

```diff
@@ -34,23 +34,24 @@
-* Table cells that exceed the height of a single page are truncated (see https://github.com/prawnpdf/prawn-table/issues/41[prawn-table#41^]).
+* Footnotes are always displayed as endnotes (at the bottom of the last page of a chapter for books; at the bottom of the last page of the document for all other doctypes).
+*Footnotes cannot be displayed at the current bottom of the page because the PDF generator does not support content reflows* (see {url-project-issues}/85#issuecomment-577412975[#85^] for reasoning).
+* Table cells that exceed the height of a single page are truncated with a warning (see https://github.com/prawnpdf/prawn-table/issues/41[prawn-table#41^]).
 * A column can't be assigned a `width` of `0%` or a `width` less than the width of a single character.
-The converter will throw the error `Prawn::Errors::CannotFit` if such a case occurs.
+The converter will skip the table and emit a warning if such a case occurs.
 * A column can't be set to `autowidth` if the width of all the other columns in the table meet or exceed 100%.
-The converter will throw the error `Prawn::Errors::CannotFit` if such a case occurs.
-* An inline image in a table cell will not force the column wider if the width of the image exceeds the width of the column; either reduce the image width using `pdfwidth`, increase the width of the column using `cols`, or convert the cell to an AsciiDoc table cell and, preferably, use a block image (see {url-project-issues}/830[#830^]).
-* Must use development version of prawn for error to include font name when requested font style is missing.
-* AsciiDoc table cell leaves padding below last block (due to lack of margin collapsing).
+The converter will skip the table and emit a warning if such a case occurs.
+* An inline image in a table cell will shrink to fit (rather than force the column wider) if the width of the image exceeds the width of the column; you can increase the width of the column using `cols` or convert the cell to an AsciiDoc table cell and, preferably, use a block image (see {url-project-issues}/830[#830^]).
+* An inline image with a percentage `width` value in an `autowidth` table cell is resized relative to its intrinsic width.
+The space reserved for the image matches its intrinsic width.
+This matches the behavior of HTML.
 * Prawn does not support double-wide box drawing glyphs correctly, so box drawings aren't aligned properly in verbatim blocks (see https://github.com/prawnpdf/prawn/issues/1002[prawn#1002^]).
-* Orphan / widow support is limited; a page break can occur between a section title and its section content, a table caption and the caption, etc.; use a manual page break to avoid.
+* Orphan / widow support is limited; for example, a page break may occur between a section title the first block of content
+ ** to avoid an orphaned section title, you can extend the converter and override the arrange_section methodarrange_section methodarrange_section methodarrange_section method
```

Comment:
I feel like something went wrong here with the "arrange_section methodarrange_section methodarrange_section methodarrange_section method"?

Reply:
:scream: